### DISCUSSION TITLE
[PSD12] Exercise 3.13

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/psd2223#181] [Esercizi (A-L)]
**Exercise 3.13** Design an asynchronously resettable D flip-flop using logic gates.

**Comment by Elia-Belli** [comment]
Per realizzare il D flip-flop con Reset Asincrono (sensibile sul fronte positivo) ho impiegato 2 D-Latches e delle porte logiche
![ARFF](https://user-images.githubusercontent.com/107944476/211653354-c65c35f5-81ff-49f3-8811-6035e5ff1d69.jpg)
E' possibile realizzarlo anche il altri modi, ad esempio modificando direttamente i D-Latch utilizzati per costruire il D flip-flop classico

**Reply by neoSnakex34** [reply]
confermo che questa soluzione funziona e che purtroppo la versione degli appunti di Simone è da correggere

**Comment by fraxb04** [comment]
@Elia-Belli 
ho controllato sugli appunti di Exyss, e lo rappresenta così:
![image](https://user-images.githubusercontent.com/122573871/214058098-a679750d-d5a5-4235-8aa9-859d4c32d8ef.png)

**Comment by Elia-Belli** [comment]
> @Elia-Belli ho controllato sugli appunti di Exyss, e lo rappresenta così: ![image](https://user-images.githubusercontent.com/122573871/214058098-a679750d-d5a5-4235-8aa9-859d4c32d8ef.png)

Non credo sia corretto, quando $R=1$ l'ingresso $CLK$ del primo D-Latch è sempre 0 quindi non viene campionato lo 0 del reset ma viene ricordato il valore precedente.
